Segfault while loading a glTF with a normal map

Hello, I'm unable to load a glTF model with a normal map in F3D v1.2.1.
 
`f3d --input='trex_s.gltf'` gives:

```
ERROR: In /__w/f3d/f3d/dependencies/vtk/Rendering/OpenGL2/vtkShaderProgram.cxx, line 437
vtkShaderProgram (0x55ac9ed42f70): 1: #version 150
2: #ifndef GL_ES
3: #define highp
4: #define mediump
5: #define lowp
6: #endif // GL_ES
7: #define attribute in
8: #define varying out
9: 
10: 
11: /*=========================================================================
12: 
13:   Program:   Visualization Toolkit
14:   Module:    vtkPolyDataVS.glsl
15: 
16:   Copyright (c) Ken Martin, Will Schroeder, Bill Lorensen
17:   All rights reserved.
18:   See Copyright.txt or http://www.kitware.com/Copyright.htm for details.
19: 
20:      This software is distributed WITHOUT ANY WARRANTY; without even
21:      the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
22:      PURPOSE.  See the above copyright notice for more information.
23: 
24: =========================================================================*/
25: 
26: in vec4 vertexMC;
27: 
28: 
29: 
30: 
31: // frag position in VC
32: out vec4 vertexVCVSOutput;
33: 
34: // optional normal declaration
35: //VTK::Normal::Dec
36: 
37: // extra lighting parameters
38: //VTK::Light::Dec
39: 
40: // Texture coordinates
41: in vec2 tcoord;
42: out vec2 tcoordVCVSOutput;
43: 
44: uniform mat4 tcMatrix;
45: 
46: // material property values
47: //VTK::Color::Dec
48: 
49: // clipping plane vars
50: //VTK::Clip::Dec
51: 
52: // camera and actor matrix values
53: uniform mat4 MCDCMatrix;
54: uniform mat4 MCVCMatrix;
55: 
56: // Apple Bug
57: //VTK::PrimID::Dec
58: 
59: // Value raster
60: //VTK::ValuePass::Dec
61: 
62: // picking support
63: //VTK::Picking::Dec
64: 
65: void main()
66: {
67:   
68: 
69:   //VTK::Color::Impl
70: 
71:     tangentVCVSOutput = tangentMC;
72:   tangentVCVSOutput = normalMatrix * tangentVCVSOutput;
73: 
74: 
75:   vec4 tcoordTmp = tcMatrix*vec4(tcoord,0.0,1.0);
76: tcoordVCVSOutput = tcoordTmp.xy/tcoordTmp.w;
77: 
78: 
79:   //VTK::Clip::Impl
80: 
81:   //VTK::PrimID::Impl
82: 
83:     vec4 posMC = vertexMC;
84:   gl_Position = MCDCMatrix * posMC;
85:   vertexVCVSOutput = MCVCMatrix * posMC;
86: 
87: 
88:   //VTK::ValuePass::Impl
89: 
90:   //VTK::Light::Impl
91: 
92:   //VTK::Picking::Impl
93: 
94:   //VTK::CustomEnd::Impl
95: }

ERROR: In /__w/f3d/f3d/dependencies/vtk/Rendering/OpenGL2/vtkShaderProgram.cxx, line 438
vtkShaderProgram (0x55ac9ed42f70): 0(71) : error C1503: undefined variable "tangentVCVSOutput"
0(71) : error C1503: undefined variable "tangentMC"
0(72) : error C1503: undefined variable "tangentVCVSOutput"
0(72) : error C1503: undefined variable "normalMatrix"
0(72) : error C1503: undefined variable "tangentVCVSOutput"
```

However [attached model](https://github.com/f3d-app/f3d/files/7830443/trex_s_model.zip) is valid and loads on [glTF Report](https://gltf.report/).

![trex_s_report](https://user-images.githubusercontent.com/36248671/148585683-ffcf97be-8dea-4e81-bb5d-afad87f719bc.png)

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -14,7 +14,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        vtk_version: [a4bdbcd4b2, v9.1.0, v9.0.0]
+        vtk_version: [e6e10357dc, v9.1.0, v9.0.0]
 
     runs-on: ${{matrix.os}}
     container: ${{ matrix.os == 'ubuntu-latest' && 'ghcr.io/f3d-app/f3d-ci' || null }}

